packages/enzyme-context-react-router-3/src/enzyme-context-react-router-3.spec.tsx

```diff
@@ -22,7 +28,7 @@ describe('enzyme-context-react-router-3', () => {
 
     mount = _mount;
 
-    ({ component, history } = _mount(<ComponentWithRouter />));
+    component = _mount(<ComponentWithRouter />);
```

Comment:
Should we rename the things in test files to `wrapper` also? Seems a bit strange to do things in the docs one way and in the code a different way.

Reply:
Yeah I'm kinda torn. I prefer the name `component`, and I think it's reasonable for this library to diverge from Enzyme in its implementation, but I just want the docs to be as uncontroversial as possible, seeing as they're the first thing most folks will see.

Reply:
^ I think that is a reasonable thing to do - we want people to find it intuitive at a glance